### DISCUSSION TITLE
Grass Patch

### DISCUSSION THREAD

**Discussion by jasonjgardner** [jasonjgardner/jg-rtx#246] [Feedback]
![PBR JG screenshot](https://github.com/jasonjgardner/jg-rtx/assets/1903667/1916efe5-6af9-467e-8407-c995995dcaeb)

### Changelog
**Updated (Java)**
- Fixed pack format version
- Fixed soul lantern texture
- Added structure blocks to Java pack
- Added POM and porosity to grass and dirt textures
- Fixed normal map conversion on stone, cobblestones and gravel
- Fixed top and bottom placement on decorated pot texture
- Updated gold ore texture to match Bedrock version

**Added (Java)**
- Custom item frame
- Bamboo single leaf

**Full Changelog**: https://github.com/jasonjgardner/jg-rtx/compare/v0.32.2...v0.32.3

> ### Coming Soon
> ![PBR JG Java mod](https://github.com/jasonjgardner/jg-rtx/assets/1903667/66dc81a0-89af-4ca8-b687-52446acfdeb5)
> [Follow me on Modrinth](https://modrinth.com/user/jason.gardner) or join me in [Discord](https://discord.gg/P58yTZ4ekY) to find my upcoming mod and modpack.
> -JG

<hr /><em>This discussion was created from the release <a href='https://github.com/jasonjgardner/jg-rtx/releases/tag/v0.32.3'>Grass Patch</a>.</em>

**Comment by kouhe3** [comment]
mcpack install failed. too many folder.
